pkg/webui/lib/get-data-str.js

```diff
@@ -0,0 +1,26 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export default function getDataStr(type, object) {
+  switch (type) {
+    case 'json':
+      return getJsonDataStr(object)
+    default:
+      throw new Error('Unknown data type')
+  }
+}
+
+function getJsonDataStr(object) {
+  return `data:text/json;charset=utf-8,${encodeURIComponent(JSON.stringify(object))}`
+}
```

Comment:
```suggestion
function convertStringToDataURI(object, mimeType = 'data:application/json;charset=utf-8') {
  return `${mimeType},${encodeURIComponent(JSON.stringify(object))}`
}
```

Note `application/json` as opposed to `text/json`, which is not an official mime type.